frontend/components/layout/Header.vue

```diff
@@ -119,6 +120,10 @@ export default Vue.extend({
             type: Boolean,
             default: false,
         },
+        pageName: {
+            type: String,
+            default: '',
+        },
```

Comment:
You created a new `pageName` prop, but you
- don't provide `this.$route?.name` to it via the parent
- didn't add a new story providing this prop

If you would mark this prop as required, you would immediately see the errors that would hint that something is off